src/main/java/core/mate/academy/service/BulldozerProduce.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProduce implements MachineProducer<Bulldozer> {
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> bulldozers = new ArrayList<>();
+        bulldozers.add(new Bulldozer());
+        bulldozers.add(new Bulldozer());
```

Comment:
You could initialize the fields of your Bulldozer here since you have a constructor for this in Bulldozer.class, e.g. 
bulldozers.add(new Bulldozer("Cat", "Spain"));